### PR TITLE
Add testing step to CircleCI config.

### PR DESCRIPTION
**Pull request by Osmose**
Seeing if this is the right way to do this.

### CODE REVIEW

**Comment by Osmose** [comment]
@milescrabill Seems like it's failing on the push to dockerhub step, any ideas why?

**Comment by Osmose** [comment]
It worked!

File: .circleci/config.yml [hunk 1]
```diff
@@ -24,6 +26,27 @@ jobs:
             docker info
             docker build -t ${CIRCLE_PROJECT_REPONAME}:build .
 
+      - run:
+          name: Install Docker Compose
```
Comment (milescrabill) [inline .circleci/config.yml:30]: This works! Another option would be to switch to an image that has docker-compose installed by default. I think this is good for now.

**Review by milescrabill — APPROVED** [review]
LGTM if it works

**Pull request merged**